src/Component/Tests/Unit/FieldTypes/EnumFieldTypeTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid\Tests\Unit\FieldTypes;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Sylius\Component\Grid\DataExtractor\DataExtractorInterface;
+use Sylius\Component\Grid\Definition\Field;
+use Sylius\Component\Grid\Exception\LogicException;
+use Sylius\Component\Grid\FieldTypes\EnumFieldType;
+use Sylius\Component\Grid\FieldTypes\FieldTypeInterface;
+use Sylius\Component\Grid\Tests\Dummy\TestEnum;
+use Sylius\Component\Grid\Tests\Dummy\TestNonBackedEnum;
+use Sylius\Component\Grid\Tests\Dummy\TestTranslatableEnum;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+final class EnumFieldTypeTest extends TestCase
+{
+    private DataExtractorInterface|MockObject $dataExtractorMock;
+
+    private EnumFieldType $enumFieldType;
+
+    protected function setUp(): void
+    {
+        $this->dataExtractorMock = $this->createMock(DataExtractorInterface::class);
+        $this->enumFieldType = new EnumFieldType($this->dataExtractorMock, $this->createMock(TranslatorInterface::class));
+    }
+
+    public function testAGridFieldType(): void
+    {
+        $this->assertInstanceOf(FieldTypeInterface::class, $this->enumFieldType);
+    }
+
+    public function testUsesDataExtractorToObtainDataAndRendersIt(): void
+    {
+        /** @var Field|MockObject $fieldMock */
+        $fieldMock = $this->createMock(Field::class);
```

Comment:
I think you can use the real instance instead.
EnumField::create...

Reply:
Sure, I now based it on the other unit tests from https://github.com/Sylius/SyliusGridBundle/pull/399/files#diff-57fa8d133cf70faa3b036d9b7d5a703885049c50865b1b6db255fd43b25031ddR47

But I've updated the tests to reflect this!